nestjs/app/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "sample-app",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```

Comment:
Can we change this to be apache? I recall having a conversation about these with Maurice recently, and we decided on sticking with what the agent uses

Reply:
Ah, oops, I forgot to modify this part of the file and just left the defaults. Will fix.